packages/data-format/src/format/rnf_invoice/rnf_invoice-0.0.2.json

```diff
@@ -133,6 +133,9 @@
           "taxPercent": {
             "type": "number"
           },
+          "taxAmount": {
```

Comment:
open question: should we leave the taxPercent a required field with this new one?

Reply:
I agree, we have to handle it better. If possible, having both being mutualy exclusive and mandatory could work.
Another option would be having a `taxAmount` field, and a `taxType` field, where we specify if it's percentage or amount.